doc/hardware/porting/board_porting.rst

```diff
@@ -8,15 +8,69 @@ directory* with various files in it. Files in the board directory inherit
 support for at least one SoC and all of its features. Therefore, Zephyr must
 support your :term:`SoC` as well.
 
+.. _board_and_identifiers
+
+Board and board identifiers
+***************************
+
+A board may be a physical piece of hardware or an emulated board.
+Furthermore a board may contain just a single SoC or it may contain multiple
+SoCs, as well as each SoC may be contain a single or multiple CPU cores.
+
+A board may further define multiple variants for various use-cases.
+Examples of such use-cases are:
+- Variant which enables non-secure builds for SoCs containing a security
+  processor.
+- Variant enabling / changing the type of RAM used in by the build.
+
+A ``/`` is used as separator between the board name and the following
+SoC, cpu core, variant identifiers.
+
+If a board contains a single SoC, then the SoC can be omitted when building.
+
+Let's say there is a board named ``plank`` with a single-core SoC ``socA``.
+The board including the identifier is: ``plank/socA``.
```

Comment:
`soc1` would be better, `socA` gives the wrong impression with camel case when it should be snake case

Reply:
Agreed and done.
All occurrences of `socA` updated to `soc1`.